apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -104,3 +126,25 @@ function buildState(steps: Record<string, unknown> | undefined): FrameworkPrevio
 
   return outputArray;
 }
+export class GeneratePreviewError extends InternalServerErrorException {
+  constructor(error: FrameworkError) {
+    super({
+      message: `GeneratePreviewError: Original Message:${error.response.message}`,
+      code: error.response.code,
```

Comment:
what do you think about adding this as params so we have a clean message as we discussed before?

`raw` to `origianlMessage`
```suggestion
      message: `GeneratePreviewError`,
      raw: error.response.message
```